### ISSUE TITLE
Bump in Garden: ign-common and dependents

### ISSUE THREAD

**Issue by chapulina** [ignition-tooling/release-tools#567]
This should come after #545, it could be combined with #554, which hasn't started yet

Note that if 

Libraries being bumped:

* `ign-common5`: DEM support (https://github.com/ignitionrobotics/ign-common/pull/267)
    - [ ] Source code of all downstream libraries ([example](https://github.com/ignitionrobotics/ign-transport/pull/149))
    - [ ] Release repositories of all downstream libraries ([example](https://github.com/ignition-release/ign-fuel-tools7-release/pull/1))
    - [ ] Use `main` branch on `gazebodistro` ([example](https://github.com/ignition-tooling/gazebodistro/pull/42))
    - [ ] Build nightlies from the `main` branch ([example](https://github.com/ignition-tooling/release-tools/pull/437))
    - [ ] Update gzdev to use nightlies on CI ([example](https://github.com/ignition-tooling/gzdev/pull/31))
    - [ ] homebrew-simulation: create formula and update dependencies ([example](https://github.com/osrf/homebrew-simulation/pull/14230))
    - [ ] docs (the collection’s page) ([example](https://github.com/ignitionrobotics/docs/pull/175))

* `ign-physics6`: Depends on `ign-common`
    - [ ] Source code of all downstream libraries ([example](https://github.com/ignitionrobotics/ign-transport/pull/149))
    - [ ] Release repositories of all downstream libraries ([example](https://github.com/ignition-release/ign-fuel-tools7-release/pull/1))
    - [ ] Use `main` branch on `gazebodistro` ([example](https://github.com/ignition-tooling/gazebodistro/pull/42))
    - [ ] Build nightlies from the `main` branch ([example](https://github.com/ignition-tooling/release-tools/pull/437))
    - [ ] Update gzdev to use nightlies on CI ([example](https://github.com/ignition-tooling/gzdev/pull/31))
    - [ ] homebrew-simulation: create formula and update dependencies ([example](https://github.com/osrf/homebrew-simulation/pull/14230))
    - [ ] docs (the collection’s page) ([example](https://github.com/ignitionrobotics/docs/pull/175))

Downstream libraries affected:

* `ign-fuel-tools8`
* `ign-gazebo7`
* `ign-gui7`
* `ign-launch6`
* `ign-rendering7`
* `ign-sensors7`
* `sdformat12` or `sdformat13` - depends on whether https://github.com/ignitionrobotics/sdformat/pull/736 introduces `ign-common` as a dependency

**Comment by chapulina** [comment]
Closing in favor of #574, it's easier to do all libraries at once.
